src/UI/interfaces.d.ts

```diff
@@ -4,6 +4,7 @@
  */
 interface TooltipItemRenderer {
   id(): string;
-  secondaryText(subtype: string): string;
-  primaryText(subtype: string): string;
+  labelMedium(subtype: string): string;
```

Comment:
These names are to match the way these are referred to in the designs - I'm not 100% fond of this, but it's an easy change later